helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -83,3 +83,28 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: LokiNeedsToBeScaledDown
+      annotations:
+        description: 'Loki is consuming very few resources and needs to be scaled down.'
+        opsrecipe: loki/
```

Comment:
I see no mention of `LokiNeedsToBeScaledDown` in the ops-recipe linked here. Please add it.

Reply:
I know, that's why I specified in my PR comment that "I will also update the loki ops-recipe to help the oncallers on how to investigate quickly." but prior to that, I wanted to have your opinion on the approach I proposed, because if you prefer to have a single alert for `loki-write` only or one alert per loki component, the ops recipe and query will be different.

Reply:
I am fine with the query and grouping all components altogether.